### PR TITLE
Added Car component which calculates certain race data (e.g. sectors)

### PR DESCRIPTION
**Pull request by patrickswijgman**

### CODE REVIEW

**Comment by DNiemeijer** [comment]
Fix the lowercase variables

File: front-end/src/components/Car.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        sectorThreeTime = Helper.secondsToStr(sectorThreeTime);
+        gap = Helper.secondsToStr(gap);
+
+        // Check if in pits
+        let inPits = "";
+        if (data.in_pits === 1) inPits = "Pitting";
+        if (data.in_pits === 2) inPits = "In Pit Area";
+
+        // Initial state
+        this.state = {
```
Comment (DNiemeijer) [inline front-end/src/components/Car.js:48]: Why is this mixed; eg sometimes a var and sometimes from the data objec, could be always vars or always data object right?

Comment (DNiemeijer) [reply front-end/src/components/Car.js:48]: These keys could be camelcased as well right?

**Review by DNiemeijer — CHANGES_REQUESTED** [review]

File: front-end/src/components/Car.js (hunk 1 above)
Comment (ghost) [reply front-end/src/components/Car.js:48]: > Why is this mixed; eg sometimes a var and sometimes from the data objec, could be always vars or always data object right?

Sometimes raw data can be shown directly and some data needs to be edited (lap time in seconds to a readable time format)

> These keys could be camelcased as well right?

Variables are now camelcased

**Pull request merged**